evalml/pipelines/component_graph.py

```diff
@@ -203,7 +203,7 @@ def fit_features(self, X, y):
             y (pd.Series): The target training data of length [n_samples].
 
         Returns:
-            pd.DataFrame: Transformed values.
+            Tuple: pd.DataFrame, pd.Series: Transformed features and target.
```

Comment:
Thank you :joy:

I'm not sure if this intended but it looks like sphinx / napoleon separates the return type only on the first colon: https://feature-labs-inc-evalml--2780.com.readthedocs.build/en/2780/autoapi/evalml/pipelines/index.html#evalml.pipelines.ComponentGraph.fit_features

If we do Tuple(pd.DataFrame, pd.Series) (unclear of exact syntax, might be square brackets heh), we could get it all on the return type line :)

Reply:
Thank you for letting me know!